src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  // static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+    this.hidden = false;
+  }
```

Comment:
Pay attention to logical order
```suggestion
  constructor(name, health = 100) {
    this.name = name;
    this.health = health;
    this.hidden = false;
    
    Animal.alive.push(this);
  }
```

Reply:
done